flocker/node/agents/blockdevice.py

```diff
@@ -944,6 +944,42 @@ def get_device_path(blockdevice_id):
         """
 
 
+class IProfiledBlockDeviceAPI(IBlockDeviceAPI):
+    """
+    An interface for ``IBlockDeviceAPI`` implementers that are capable of
+    creating volumes with a specific profile.
+    """
+
```

Comment:
Sticking as is for now. Will re-visit before we have people outside the company try to implement one of these.